app/models/License.php

```diff
@@ -156,6 +156,16 @@ public function supplier()
         return $this->belongsTo('Supplier','supplier_id');
     }
 
+public function freeSeat()
```

Comment:
I think we'd want to use the `LicenseSeat` model (versus DB table calls directly) so we can eager load relatioonships, etc